setup.py

```diff
@@ -7,7 +7,7 @@
 import sys
 from setuptools import setup, find_packages
 
-LIBRARY_VERSION = '1.1.1'
+LIBRARY_VERSION = '1.1.4'
```

Comment:
Unfortunately this is still manual: looks like we forgot to update the version for 1.1.2 and 1.1.3